modules/contact/src/constraints/ComputeWeightedGapLMMechanicalContact.C

```diff
@@ -206,6 +206,8 @@ ComputeWeightedGapLMMechanicalContact::computeJacobian(const Moose::MortarType m
 void
 ComputeWeightedGapLMMechanicalContact::post()
 {
+  parallel_object_only();
```

Comment:
We should also plan on extending this call to other mortar objects in the future, shouldn't we?

Reply:
yes

Reply:
IMHO there's probably a thousand places in MOOSE that would benefit from this call.  I figured for now I'd just leave this reactively-added batch in and see if there was much interest or much pushback before I considered getting proactive about it.